be/src/exec/tablet_sink.cpp

```diff
@@ -849,22 +865,37 @@ Status OlapTableSink::_init_node_channels(RuntimeState* state) {
     const auto& partitions = _vectorized_partition->get_partitions();
     for (int i = 0; i < _schema->indexes().size(); ++i) {
         // collect all tablets belong to this rollup
-        std::vector<TTabletWithPartition> tablets;
```

Comment:
What's the difference between TTabletWithPartiton and PTabletWithPartition ?

Reply:
no difference